cmake/compiler/clang.cmake

```diff
@@ -9,6 +9,16 @@ if(NOT DEFINED NOSYSDEF_CFLAG)
   set(NOSYSDEF_CFLAG -undef)
 endif()
 
+set(CMAKE_C_COMPILER   ${CLANG_ROOT}/bin/clang)
+set(CMAKE_CXX_COMPILER ${CLANG_ROOT}/bin/clang++)
+set(CMAKE_AR           ${CLANG_ROOT}/bin/llvm-ar		CACHE INTERNAL " " FORCE)
+set(CMAKE_LINKER       ${CLANG_ROOT}/bin/llvm-link		CACHE INTERNAL " " FORCE)
+set(CMAKE_NM           ${CLANG_ROOT}/bin/llvm-nm		CACHE INTERNAL " " FORCE)
+set(CMAKE_OBJDUMP      ${CLANG_ROOT}/bin/llvm-objdump	CACHE INTERNAL " " FORCE)
```

Comment:
nit: alignment. seems to be tab/space mixed.

Reply:
Fixed in https://github.com/zephyrproject-rtos/zephyr/pull/11810